esphome/components/ds3231/__init__.py

```diff
@@ -0,0 +1,20 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import (CONF_ID)
+
+DEPENDENCIES = ['i2c']
```

Comment:
@snakeye You need to add this here and then run `script/build_codeowners.py`

```suggestion
CODEOWNERS = ['@snakeye']
DEPENDENCIES = ['i2c']
```